### DISCUSSION TITLE
Override z-index of multi-select box

### DISCUSSION THREAD

**Discussion by Uzeyir9695** [primefaces/community#354] [PrimeVue] [answered]
Hello, I am posting second time because my previous post wasn't answered. Hope this time I will get answer.
So how can I override multi-select box's drop-down z-index which has class name ".p-multiselect-panel" ? It always takes high priority despite of trying even setting inline z-index.

**Comment by FlipWarthog** [comment]
Did you try setting the `panelStyle` prop on MultiSelect? `panelStyle="z-index: 2002;"`

> Name: panelStyle
> Description: Inline style of the overlay panel.

https://primevue.org/multiselect/#api.multiselect.props.panelStyle

**Reply by Uzeyir9695** [reply]
Thanks a lot for the hint. panelStyle didin't work but panelClass worked.

**Comment by tugcekucukoglu** [comment, marked as answer]
Hello,

There is also a global solution if you prefer.  Default value for overlay components is 1000. 

```
import {createApp} from "vue";
import PrimeVue from "primevue/config";
const app = createApp(App);

app.use(PrimeVue, {
    zIndex: {
        overlay: 1000,      //dropdown, multiselect
    }
});
```
